os/fs/inode/fs_inodereserve.c

```diff
@@ -195,6 +197,12 @@ int inode_reserve(FAR const char *path, FAR struct inode **inode)
 		return -EINVAL;
 	}
 
+	/* Check file name length */
+
+	if (strlen(path) > NAME_MAX) {
```

Comment:
CONFIG_NAME_MAX means just file name, not including path.
PATH_MAX including everything.

Reply:
Modified to check only the length of file names.